requirements.txt

```diff
@@ -7,3 +7,4 @@ argparse==1.1
 fastchardet==0.2.0
 mock==1.0.0
 simplejson==3.8.0
+spidermonkey==41.0a2.post1
```

Comment:
Yeah, we'll need it there. Do you want me to give you a wheel for Linux? PyPI won't accept it.